frontend_next/app/(root)/case/[slug]/page.tsx

```diff
@@ -0,0 +1,115 @@
+import Image from "next/image";
+import Link from "next/link";
+
+import {
+  TechStack,
+  Info,
+  Process,
+  Gains,
+  RelevantCaseStudies,
+} from "@/components/index";
+
+export default function CaseStudy() {
+  return (
+    <>
+      <section className="bg-1 space-y-6 text-center">
+        <h2 className="uppcase text-accent-primaryLight md:p-bold dark:text-accent-primaryDark s-bold leading-[130%] tracking-[4.2px] md:tracking-[6px]">
+          Web Dev Project
+        </h2>
+
+        <h1 className="pageTitle">Morrent - A Car Rental Website</h1>
+
+        <Image
+          src="/assets/morent.svg"
+          alt="project image"
+          width={0}
+          height={0}
```

Comment:
It's important for the browser when rendering the image to know the size of the images. Have you ever been on a website and as its loading all the elements are shifting around and moving? This can be a cause of that. Set the correct dimensions so the browser knows what space to make for the element as it loads in.